<launch>
    <node name="LidarNode" pkg="uahr_lidar" type="uahr_lidar_node" output="screen"/>    
    <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
    <param name="serial_baudrate"     type="int"    value="115200"/><!--A1/A2 -->
    <param name="frame_id"            type="string" value="laser"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="pose_x"              type="int"    value="1430"/>
    <param name="pose_y"              type="int"    value="0"/>
    <param name="pose_alfa"           type="int"    value="180"/>
    <param name="angle_zero_lidar_front"    type="int"   value="180"/>

    <rosparam command="load"          file="$(find uahr_lidar)/config/yamls/coordenadas_amarillo.yaml"/>
    <param name="modo"                type="string"     value="demo"/>
</launch>